BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -319,13 +356,17 @@ public final class ConvivaAnalytics: NSObject {
             return
         }
 
+        if isSsaiAdBreakActive {
+            adAnalytics.reportAdEnded()
+            videoAnalytics.reportAdBreakEnded()
+        }
```

Comment:
NIT:
Is that really needed? 🤔 It's also not done for CSAI

Reply:
It is how we do it on Android as well.
`reportAdEnded()` is documented that it has to be called on error.

Interestingly `reportAdBreakEnded()` is not mentioned there 🤔 @strangesource what do you think? should we remove `reportAdBreakEnded()` here?

ref:
<img width="671" alt="Screenshot 2024-07-04 at 14 23 55" src="https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/assets/2691371/be6964e6-0196-4017-964f-8d81993c757d">

Reply:
Hard to tell, I don't think there is clear evidence for one or the other to be honest. 🤔 If we end the session anyways it probably doesn't matter.
Let's keep it.